components/cloud/azure/src/azblob.rs

```diff
@@ -225,47 +236,35 @@ impl BlobConfig for Config {
 
 enum RequestError {
     InvalidInput(Box<dyn std::error::Error + Send + Sync>, String),
-    InternalError(String),
```

Comment:
the new azure sdk supports the retry backoff, so no need to wrap the another retry anymore.